### PR TITLE
Closes Issue #70 - adding toggleable layers

### PR DESCRIPTION
**Pull request by ltmetz**

### CODE REVIEW

File: src/app/sidebar-left/sidebar-left.component.html [hunk 1]
```diff
@@ -37,23 +37,44 @@ <h3 class="sidebar-popout-header">Base Maps</h3>
...
 				
+				<!-- Section for layers that load automatically/not part of an active workflow  -->
+				<div  *ngIf="overlays.length">
+					<h4>Base Layers</h4>
+					<div *ngFor="let overlay of overlays">
+						
+						<div *ngIf="overlay[0] === 'Streamgages'">
+							<input #overlayCheckbox type="checkbox" id="{{overlay[0]}}" (change)="setOverlayLayer(overlay[0])" [checked]="currentZoom >= 8" [disabled]=" 8 > currentZoom"> 
+							{{overlay[0]}}
+							<label  style="font-size: small;">(active at zoom level {{overlay[1]}} or greater)</label>
```
Comment (amedenblik) [inline src/app/sidebar-left/sidebar-left.component.html:60]: We can omit the text "(active at zoom level X or greater)"-- I think we can rely on the disabled checkboxes based on zoom level

File: src/app/sidebar-left/sidebar-left.component.html [hunk 2]
```diff
@@ -37,23 +37,44 @@ <h3 class="sidebar-popout-header">Base Maps</h3>
...
+							{{overlay[0]}}
+						</div>
+
+					</div>
+				</div>
+				<!-- Section for layers that part of teh active workflow  -->
+				<div *ngIf="workflowLayers.length">
+					<h4>Active Workflow Layers</h4>
+					<div *ngFor="let layer of workflowLayers">
+						<input type="checkbox" name="activeWorkflowLayers" value="layer.name" (change)="updateActiveLayer(layer.name)" [checked]="true"> {{layer.name}}
```
Comment (amedenblik) [inline src/app/sidebar-left/sidebar-left.component.html:74]: Some of the workflow layers have a minZoom requirement (example: Burn Severity has minZoom 10), so we want to disable those checkboxes when applicable too.

File: src/app/sidebar-left/sidebar-left.component.html [hunk 3]
```diff
@@ -37,23 +37,44 @@ <h3 class="sidebar-popout-header">Base Maps</h3>
...
-				This is the content of the map layers button popout window.
-				<br/>
-				Show and hide the map layers button with a click on the button.sidebar-button
-				<br/><br/><br/><br/><br/><br/>
-				If you have too much content in the map layers, it'll scroll.
-
 				
+				<!-- Section for layers that load automatically/not part of an active workflow  -->
+				<div  *ngIf="overlays.length">
+					<h4>Base Layers</h4>
```
Comment (amedenblik) [inline src/app/sidebar-left/sidebar-left.component.html:54]: Rename "Base Layers" to "Application Layers" as to not cause confusion with Base Maps.

**Review by amedenblik — CHANGES_REQUESTED** [review]
Awesome work! This will be super helpful! Some things I noticed:

- [x] When I uncheck layers, click the "X" to close the Map Layers popout, and click the icon to open the Map Layers popout again, all the layers are checked again.  

- [ ] Allow the user to click on the name of the layer to check/uncheck the checkbox. You can do this by included the <input /> field inside the <label></label> tags. Or, you can do this by using the "for" attribute on the <label> tag. 

- [ ] When I click Report Builder and select a Fire Hydro workflow, the workflow layers are added to the Map Layers popout. Then I click the "X" to close the Report Builder popout, and the workflow has been exited out of, but that workflow's map layers are still in the Map Layers popout. So if I again choose a Fire Hydro workflow, now two copies of the Fire Hydro workflow layers appear in the Map Layers popout. I think this is more of an issue that clicking "X" to close the Report Builder popout shouldn't exit out of the current workflow-- see issue #88.

**Comment by ltmetz** [comment]
Third checkmark is now issue #100

**Review by amedenblik — APPROVED** [review]
Confirmed that comments were addressed ✅

**Pull request merged**